telemetry/definitions/commonDefinitions.json

```diff
@@ -3145,6 +3190,80 @@
                 { "type": "codecatalyst_devEnvironmentWorkflowStep"},
                 { "type": "codecatalyst_devEnvironmentWorkflowError", "required": false }
             ]
+        },
+        {
+            "name": "codewhispererterminal_userLoggedIn",
+            "description": "Emitted when users log in",
```

Comment:
is this only emitted when user successfully logs in? do we have any metrics if user attempted to log in? you can return existing telemetry metrics too. 
e.g. there's credentialModification 
 {
            "name": "credentialModification",
            "type": "string",
            "allowedValues": ["Add", "Edit", "Delete"],
            "description": "The type of modification performed on the credentials"
        }